Dockerfile.aaq-controller

```diff
@@ -1,33 +1,37 @@
 # Builder stage
-FROM docker.io/library/golang:1.22-alpine as builder
+FROM  --platform=${BUILDPLATFORM} docker.io/library/golang:1.23-alpine as builder
+ARG BUILDPLATFORM
```

Comment:
Shouldn't the arg be declared before it is used?

Reply:
Not in `FROM`.

Reply:
And BTW, this ARG is builtin in podman/docker

Reply:
Yes, i meant to add the `ARG` before the `FROM` line. That is more a syntax question than a function one. If it is wrong you should only get a warning.